Neos.Neos/Classes/Controller/Backend/MenuHelper.php

```diff
@@ -81,34 +89,38 @@ public function buildSiteList(ControllerContext $controllerContext): array
             return [];
         }
 
+        $context = $this->contextFactory->create();
         $domainsFound = false;
         $sites = [];
         foreach ($this->siteRepository->findOnline() as $site) {
-            $uri = null;
-            $active = false;
-            /** @var $site Site */
-            if ($site->hasActiveDomains()) {
-                $activeHostPatterns = $site->getActiveDomains()->map(static function ($domain) {
-                    return $domain->getHostname();
-                })->toArray();
-
-                $active = in_array($requestUriHost, $activeHostPatterns, true);
-
-                if ($active) {
-                    $uri = $contentModule['uri'];
-                } else {
-                    $uri = $controllerContext->getUriBuilder()->reset()->uriFor('switchSite', ['site' => $site], 'Backend\Backend', 'Neos.Neos');
+            $node = $context->getNode("/sites/" . $site->getNodeName());
```

Comment:
Are site nodes _guaranteed_ to be underneath `/sites/`?
If so we should still replace the string with a constant (there is `SiteService::SITES_ROOT_PATH` IIRC) or at least replace the double quotes :)

Reply:
Yep, `Neos\Neos\Domain\Service\SiteService::SITES_ROOT_PATH` is set to `/sites`.
```suggestion
            $node = $context->getNode(\Neos\Neos\Domain\Service\SiteService::SITES_ROOT_PATH . '/' . $site->getNodeName());
```
(and please add an import for the `SiteService`)

Reply:
@kdambekalns that's what I was looking for... done!